### ISSUE TITLE
Both Conda and PyPi installations have conflicts for numpy versions between comp and f2py

### ISSUE THREAD

**Issue by erogluorhan** [NCAR/geocat-comp#150]
**Describe the bug**
While installing GeoCAT-comp from either conda or pip, there are version conflicts that seem to be due to Numpy versions (It looks like Numpy==1.19 was fogotten in `meta.yaml` of GeoCAT-comp and `requirements.txt` of both GeoCAT-comp and GeoCAT-f2py).

**To Reproduce**
1. Create a new conda environment
2. Either run `conda install -c ncar geocat-comp`, or
3. Run `pip install geocat-comp`

**Expected behavior**
GeoCAT-comp v2021.05.0 should have been installed with the dependency of GeoCAT-f2py and other packages (Xarray etc.) also being installed automatically.

**Screenshots**
![image](https://user-images.githubusercontent.com/32553057/116974841-f79abc00-ac7b-11eb-80a3-302e647ca9e3.png)

![image](https://user-images.githubusercontent.com/32553057/116974913-11d49a00-ac7c-11eb-8782-597c7142cabf.png)

**OS:**
 MacOS

**Environment**
```
(geocat_pip_trial) Orhans-MacBook-Pro:~ orhaneroglu$ conda list
# packages in environment at /Applications/anaconda3/envs/geocat_pip_trial:
#
# Name                    Version                   Build  Channel
ca-certificates           2021.4.13            hecd8cb5_1  
certifi                   2020.12.5        py39hecd8cb5_0  
libcxx                    10.0.0                        1  
libffi                    3.3                  hb1e8313_2  
ncurses                   6.2                  h0a44026_1  
openssl                   1.1.1k               h9ed2024_0  
pip                       21.0.1           py39hecd8cb5_0  
python                    3.9.4                h88f2d9e_0  
readline                  8.1                  h9ed2024_0  
setuptools                52.0.0           py39hecd8cb5_0  
sqlite                    3.35.4               hce871da_0  
tk                        8.6.10               hb0a8c7a_0  
tzdata                    2020f                h52ac0ba_0  
wheel                     0.36.2             pyhd3eb1b0_0  
xz                        5.2.5                h1de35cc_0  
zlib                      1.2.11               h1de35cc_3
```

**Comment by pilotchute** [comment]
on it

**Comment by pilotchute** [comment]
Fixed with release v2021.05.1
